File: examples/java/src/main/java/org/example/objectstore/Objectstore.java
package org.example.objectstore;

import com.amazonaws.auth.AWSCredentials;
import com.amazonaws.auth.BasicAWSCredentials;
import com.amazonaws.services.s3.AmazonS3;

public class Objectstore {
  private static final String USERNAME = "minioadmin";
  private static final String PASSWORD = "minioadmin";
  private static final String URL = "http://localhost:9000";
  private static final String BUCKET_NAME = "test";
  private static final String FILE_NAME = "file";

  public static void main(String[] args) {
    AWSCredentials credentials = new BasicAWSCredentials(USERNAME, PASSWORD);
    AmazonS3 s3 = Operations.getConnection(URL, credentials, true);

    Operations.demo(s3, BUCKET_NAME, FILE_NAME);
  }
}

File: examples/python/objectstore.py
import boto3
import urllib3

from botocore.client import Config
from operations import demo


if __name__ == '__main__':
    host = "http://localhost:9000"
    username = "minioadmin"
    password = "minioadmin"
    bucketName = "test"
    file = "file"

    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
    s3 = boto3.client('s3',
                      endpoint_url=host,
                      aws_access_key_id=username,
                      aws_secret_access_key=password,
                      config=Config(signature_version='s3v4'),
                      region_name='us-east-1',
                      verify=False)

    demo(s3, bucketName, file)

